WooCommerce/UITestsFoundation/Screens/ExternalAppScreen.swift

```diff
@@ -31,7 +31,7 @@ public final class ExternalAppScreen {
         safari.buttons["Go"].tap()
 
         // Tap on the universal link and fail test immediately if link is not hittable
```

Comment:
The comment should be updated, too, now that we are waiting for the link to be hittable:

```suggestion
        // Tap on the universal link and fail test if link is not hittable
```

For what is worth, I think the code below is self-describing, and would be happy removing the comment, instead:

```suggestion
```

Reply:
removed in https://github.com/woocommerce/woocommerce-ios/pull/9560/commits/736acadda8795a916b4561ddc022fe9791fd1611